azurerm/resource_arm_application_gateway.go

```diff
@@ -517,6 +522,11 @@ func resourceArmApplicationGateway() *schema.Resource {
 							Required: true,
 						},
 
+						"pick_host_name_from_backend_http_settings": {
+							Type:     schema.TypeBool,
+							Optional: true,
+						},
```

Comment:
we should explicitly show the default as `false`

Reply:
done